Userland/Libraries/LibWeb/HTML/DOMParser.cpp

```diff
@@ -25,7 +25,7 @@ NonnullRefPtr<DOM::Document> DOMParser::parse_from_string(String const& string,
     // 2. Switch on type:
     if (type == Bindings::DOMParserSupportedType::Text_Html) {
         // -> "text/html"
-        // FIXME: 1. Set document's type to "html".
```

Comment:
Keep `// 1. Set document's type to "html".`, remove just the `FIXME` part of the comment.